pkg/features/embedded/elemental-rootfs/usr/lib/dracut/modules.d/30elemental-rootfs/module-setup.sh

```diff
@@ -24,15 +24,10 @@ install() {
     declare systemdsystemunitdir=${systemdsystemunitdir}
 
     inst_multiple \
-        mount mountpoint sort rmdir findmnt rsync cut realpath basename lsblk
```

Comment:
Since there is the `elemental mount` command this dracut module doesn't require most of the pieces it used to as most of the logic simply moved to `elemental mount`.